### ISSUE TITLE
Throwing error when requesting

### ISSUE THREAD

**Issue by OceanTechnic** [Linux123123/JSPteroAPI#74]
Error trace
```
npm run server[25984]: c:\ws\src\tcp_wrap.cc:155: Assertion `args[0]->IsInt32()' failed.
1: 00007FF6ED25151F v8::internal::CodeObjectRegistry::~CodeObjectRegistry+121999
2: 00007FF6ED1DB386 DSA_meth_get_flags+64118
3: 00007FF6ED1DB741 DSA_meth_get_flags+65073
4: 00007FF6ED0F220F v8::internal::HashTable<v8::internal::GlobalDictionary,v8::internal::GlobalDictionaryShape>::HashTable<v8::internal::GlobalDictionary,v8::internal::GlobalDictionaryShape>+23039
5: 00007FF6EDACD656 v8::internal::Builtins::code_handle+172806
6: 00007FF6EDACCECE v8::internal::Builtins::code_handle+170878
7: 00007FF6EDACD4F7 v8::internal::Builtins::code_handle+172455
8: 00007FF6EDACD370 v8::internal::Builtins::code_handle+172064
9: 00007FF6EDBA0A51 v8::internal::SetupIsolateDelegate::SetupHeap+494641
10: 00007FF6EDB30506 v8::internal::SetupIsolateDelegate::SetupHeap+34534
11: 00007FF6EDC30476 v8::internal::SetupIsolateDelegate::SetupHeap+1082966
12: 00007FF6EDB32F1E v8::internal::SetupIsolateDelegate::SetupHeap+45310
13: 0000016E1EF1BDC8
```
Node version 16 and higher
Version 14 will throw fetch is not a function in udici

Maybe somehow related?
https://github.com/nodejs/node/issues/35572
https://github.com/nodejs/undici/issues/448

**Comment by Linux123123** [comment]
https://github.com/nodejs/undici#undicifetchinput-init-promise

I believe `undici.fetch` method is only available in Node 16.8+. And it is used to request. I remember I tried to implement the `undici.request` method, but never successfully did, as it doesn't have some features that `fetch` does.
https://github.com/Linux123123/JSPteroAPI/blob/ce44eae3c52290b28c9f5957750fc84be0d22e5b/src/modules/Request.ts#L1

I will update `engines` property of `package.json` accordingly.

**Comment by OceanTechnic** [comment]
Cool I see your changes https://github.com/Linux123123/JSPteroAPI/commit/eba333e3455a0dd8cef4b70f0179a15861c89fe3, let me check with my code
